Update expected test results for libheif 1.18 and newer.

Changes guided by portions of
https://github.com/Cykooz/libheif-rs/commit/8b3b39976badf38187418bb3547ad7754c7736d3

Index: libheif-rs/tests/read_test.rs
===================================================================
--- libheif-rs.orig/tests/read_test.rs
+++ libheif-rs/tests/read_test.rs
@@ -50,11 +50,11 @@ fn read_from_reader() -> Result<()> {
 
     ctx.set_max_decoding_threads(2);
     let preferred_colorspace = handle.preferred_decoding_colorspace().unwrap();
-    assert_eq!(preferred_colorspace, ColorSpace::YCbCr(Chroma::C444));
+    assert_eq!(preferred_colorspace, ColorSpace::YCbCr(Chroma::C420));
     let src_img = lib_heif.decode(&handle, ColorSpace::Undefined, None)?;
     assert_eq!(
         src_img.color_space(),
-        Some(ColorSpace::Rgb(RgbChroma::C444)) // Hmm... It doesn't equal to preferred_colorspace. It's weird.
+        Some(ColorSpace::YCbCr(Chroma::C420)) // Hmm... It doesn't equal to preferred_colorspace. It's weird.
     );
 
     Ok(())
@@ -217,12 +217,12 @@ fn top_decode_heic() -> Result<()> {
     let src_img = lib_heif.decode(&handle, ColorSpace::Undefined, None)?;
     assert_eq!(
         src_img.color_space(),
-        Some(ColorSpace::Rgb(RgbChroma::C444))
+        Some(ColorSpace::YCbCr(Chroma::C420))
     );
     let planes = src_img.planes();
-    let r_plane = planes.r.unwrap();
-    assert_eq!(r_plane.width, 1652);
-    assert_eq!(r_plane.height, 1791);
+    let y_plane = planes.y.unwrap();
+    assert_eq!(y_plane.width, 1652);
+    assert_eq!(y_plane.height, 1791);
 
     Ok(())
 }
@@ -322,15 +322,15 @@ fn test_raw_color_profile_of_image() ->
         // Hmm, unspecified profile
         assert_eq!(
             nclx_profile.color_primaries(),
-            ColorPrimaries::ITU_R_BT_709_5
+            ColorPrimaries::Unspecified
         );
         assert_eq!(
             nclx_profile.transfer_characteristics(),
-            TransferCharacteristics::IEC_61966_2_1
+            TransferCharacteristics::Unspecified
         );
         assert_eq!(
             nclx_profile.matrix_coefficients(),
-            MatrixCoefficients::ITU_R_BT_601_6
+            MatrixCoefficients::Unspecified
         );
     }
     Ok(())
@@ -351,11 +351,11 @@ fn test_read_avif_image() -> Result<()>
     assert!(nclx_profile.is_some());
 
     let image = lib_heif.decode(&handle, ColorSpace::Undefined, None)?;
-    assert_eq!(image.color_space(), Some(ColorSpace::Rgb(RgbChroma::C444)));
+    assert_eq!(image.color_space(), Some(ColorSpace::YCbCr(Chroma::C444)));
     let planes = image.planes();
-    let r_plane = planes.r.unwrap();
-    assert_eq!(r_plane.width, 2048);
-    assert_eq!(r_plane.height, 1440);
+    let y_plane = planes.y.unwrap();
+    assert_eq!(y_plane.width, 2048);
+    assert_eq!(y_plane.height, 1440);
 
     Ok(())
 }
